src/mlpack/methods/kmeans/hamerly_kmeans_impl.hpp

```diff
@@ -132,6 +149,8 @@ double HamerlyKMeans<DistanceType, MatType>::Iterate(const arma::mat& centroids,
   size_t furthestMovingCluster = 0;
   arma::vec centroidMovements(centroids.n_cols);
   double centroidMovement = 0.0;
+  #pragma omp parallel for reduction(+:distanceCalculations,centroidMovement) \
+    schedule(static)
```

Comment:
```suggestion
  #pragma omp parallel for reduction(+: distanceCalculations, centroidMovement) \
      schedule(static)
```
Just some small pedantic style fixes. :+1: